app/src/main/java/com/sun/spoonacular/ui/home/HomeFragment.kt

```diff
@@ -16,7 +31,65 @@ class HomeFragment : Fragment() {
         return inflater.inflate(R.layout.fragment_home, container, false)
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initDataHome()
+
+    }
+
+    private fun initDataHome() {
+        recipes = arguments?.getParcelableArrayList<Recipe>(LIST_RECIPE_KEY) as ArrayList<Recipe>
+        for (i in 0..4) {
```

Comment:
tìm hiểu và xem xét dùng for loop nào thích hợp hơn

Reply:
> tìm hiểu và xem xét dùng for loop nào thích hợp hơn

Chỗ này em duyệt list recipe có 15 phần tử bên splash gởi qua á a, nhưng slide chỉ cần có 5 phần tử thôi, còn 10 cái sau em để cho cái recyclerview bên dưới